model/bindings.go

```diff
@@ -3,38 +3,35 @@ package model
 import (
 	"fmt"
 	log "github.com/Sirupsen/logrus"
-	yaml "github.com/cloudfoundry-incubator/candiedyaml"
 	"github.com/docker/libcompose/config"
-	"github.com/docker/libcompose/utils"
+	utils "github.com/docker/libcompose/utils"
+	libYaml "github.com/docker/libcompose/yaml"
 	"github.com/rancher/rancher-compose/preprocess"
-	"github.com/rancher/rancher-compose/rancher"
+	yaml "gopkg.in/yaml.v2"
 	"io/ioutil"
 	"os"
-	"reflect"
 )
 
-type mapLabel map[string]string
+type mapLabel map[string]interface{}
+type portArray []interface{}
 
 //BindingProperty holds bindings
 type BindingProperty map[string]interface{}
 
 //ServiceBinding holds the fields for ServiceBinding
 type ServiceBinding struct {
-	Scale  int               `json:"scale"`
-	Labels map[string]string `json:"labels"`
-	Ports  []string          `json:"ports"`
+	Scale  interface{}            `json:"scale"`
+	Labels map[string]interface{} `json:"labels"`
+	Ports  []interface{}          `json:"ports"`
 }
 
 //CreateBindingsRancher creates bindings property for rancher-compose.yml
 func CreateBindingsRancher(pathToYml string) BindingProperty {
```

Comment:
I think it would be a good idea for this function to return an error. There are a lot of places in here where we just log and fall through when an error occurs. It's probably better to immediately return rather than continuing in these scenarios. cc @prachidamle